### ISSUE TITLE
Option to use Sass Comment Style

### ISSUE THREAD

**Issue by AdrianoCahete** [styledown/styledown#19]
I need to Styledown read scss files, but i don't want to theses comments went to the final css file. All comments in my scss files are stripped with dual slashes (//).

Can i use Styledown with this type of comment?

**Comment by ryanmclaughlin** [comment]
:+1: +1

**Comment by marcelhadorn** [comment]
👍🏼+1

**Comment by mdings** [comment]
Setting output style to compressed should remove multiline comments `/* comment */` if I'm not mistaking.

**Comment by AdrianoCahete** [comment]
> Setting output style to compressed should remove multiline comments /\* comment */ if I'm not mistaking.

In SCSS, comments with dual slashes (// comment) is already removed from final file, independent of selected output. But styledown didn't read that as comment and change this in my projects (a HUGE project) isn't a option.

**Comment by rstacruz** [comment]
Hi friends,

I'm planning to build styledown 2 soon, and one of the biggest changes will be that inline CSS documentation will _not_ be supported anymore. In retrospect, it was a mistake... once your site gets big, organizing comments in CSS files gets very messy.

The existing `.md` format will continue to be supported.

Would love the community's comments on this!

**Comment by AdrianoCahete** [comment]
> once your site gets big, organizing comments in CSS files gets very messy.

I think exactly the opposite. When you have a big project, if documentation isn't easy to find/update, they'll be out of date. And the best location to find docs is inside sass files.

You can create/edit a sass file and update docs at the same time. If you move docs from outside sass file, people will stop update the docs, because many reasons (less time, eternal "todo" and etc).

But yes, if you didn't strip comments to output file, this is will be a mess, this is why original issue is to parse Sass comment style.

I hope you can change your mind. :)

**Comment by MatthieuScarset** [comment]
I agree with @AdrianoCahete 

The issue is not with inline commenting, it's just about how modular is your CSS.

For instance, if I'm about to change some styles of my site header I'm expecting to find all the doc and styles at the same place, within the SASS /modules/header.sass

This way, I can edit my styles and right new documentation at the same time.
I don't repeat myself.
It's good practice.

**You should definitely keep inline CSS documentation**

**Comment by marcelhadorn** [comment]
**I also vote for keeping the inline CSS solution**. With good organised CSS the inline docs are just as well organised. However, an option to somehow add spacers between element groups or even a anchor navigation would be fantastic!

BTW: use compressed output for production CSS, then als /**/ Comments are stripped.

**Comment by josephrexme** [comment]
Just curious ya'll. Do you run styledown against your .scss files? It says it doesn't favor preprocessors. Will it recursively look through @imports ?

**Comment by rstacruz** [comment]
Nah, it doesn't, you'll need to specify all filenames like `styledown app/css/*/*.scss` or something to that effect.

**Comment by AdrianoCahete** [comment]
> BTW: use compressed output for production CSS, then als /**/ Comments are stripped.

Rewrite all my style isn't an option. :(

**Comment by mzdr** [comment]
I'm with @AdrianoCahete, @MatthieuScarset and @Marc3llo. Please keep the inline CSS documentation!

**Comment by planktonWD** [comment]
I'm with @AdrianoCahete, @MatthieuScarset, @mzdr and @Marc3llo! I love the inline CC documentation and would never use md for documenting CSS. I use grunt-styedown and create two compiled CSS files. One is compressed and used for the site, other is uncompressed and used for styledown. I've had to create special .js and .css files for stydedown to hide CSS comments and only show the jade comments but it is working great. Love to contribute to a .scss only styledown system if V2 is on the cards.
